filebeat/module/system/audit/manifest.yml

```diff
@@ -4,7 +4,7 @@ var:
   - name: paths
     default:
       - /var/log/audit/audit.log*
-    os.darwin: []
+    os.darwin: [""]
     os.windows: []
```

Comment:
Yeah, I think it's better to leave at is, so that users get an error on windows when trying this module (otherwise, the problem would be silently ignored).